Data/Interfaces/ICustomerService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Data.Entities;
+using System.Collections.Generic;
+
+namespace Data.Interfaces
+{
+    public interface ICustomerService
+    {
+        void Create(Customer customer);
+        void Update(Customer customer);
+        void Delete(long id);
+        List<Customer> GetAll();
```

Comment:
Como quem está chamando deve apenas iterar em cima ou usar funcionalidades básicas, então na perspectiva de design sempre é preferível usar o tipo menos derivado possível (IEnumerable ou ICollection).

```suggestion
        IEnumerable<Customer> GetAll();
```